wavefront/README.md

```
@@ -1 +1,97 @@
+```
+
+Now open http://localhost:5173 in your browser to see the frontend & login with the credentials set in environment variables.
+
+**Step 6**: Go to the console, add your first app by clicking on the `Add App` button. Use the details of the floware service you started on Step 4
+
+## Next Steps
+
+Connect your datasources, and create your first agent & workflow or go ahead an try out the voice bot feature. 
+
+- Checkout the platform docs here [https://wavefront.rootflo.ai](https://wavefront.rootflo.ai/).
+- Incase you face any issues, dont hesitate to reach out to schedule a call with us [here](https://calendly.com/meetings-rootflo/30min)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)